packages/playwright-test/src/workerRunner.ts

```diff
@@ -268,7 +268,7 @@ export class WorkerRunner extends EventEmitter {
         if ((options.path !== undefined ? 1 : 0) + (options.body !== undefined ? 1 : 0) !== 1)
           throw new Error(`Exactly one of "path" and "body" must be specified`);
         if (options.path) {
-          const hash = await calculateFileSha1(options.path);
+          const hash = calculateSha1(options.path);
```

Comment:
What's the reason for the switch to filename instead of contents? Regardless, filename hash seems much simpler (and I suppose negligibly faster depending on what was in the files).

Reply:
You got it right 😄